m0t1fs/linux_kernel/file.c

```diff
@@ -1528,6 +1530,8 @@ static int pargrp_iomap_parity_recalc(struct pargrp_iomap *map)
 
 		unsigned long zpage;
 
+		rc = 0;
```

Comment:
No need to have blank lines here. The same for other pleaces.

Reply:
Done.